manifests/config.pp

```diff
@@ -50,6 +50,14 @@
     path => "${foreman_proxy::config_dir}/settings.yml",
   }
 
+  $uuid_fact_signature = foreman_proxy::hmac_signature($foreman_proxy::oauth_consumer_secret, $foreman_proxy::instance_id)
+  file { 'smart_proxy_uuid.json':
+    path    => '/opt/puppetlabs/facter/facts.d/smart_proxy_uuid.json',
```

Comment:
Is `/etc/puppetlabs/facter/facts.d` cleaner than `/opt`?

Reply:
Changed